### PR TITLE
updating-legacy-doc-images

### PR DESCRIPTION
**Pull request by zamusacelre**
- adding to routing spec
- start on screening
- lots of renaming of legacy images - cleaned out if image not used, and fixed a couple

### CODE REVIEW

File: docs/source/legacy-dev-docs.rst [hunk 1]
```diff
@@ -151,13 +151,11 @@ system you can turn pool tracing on for an application with the following java f
 
 .. code-block:: bash
 
-    DECJ_MAXHEAP="-Dopendcs.connection.pool.trace=true" routsched ...
```
Comment (MikeNeilson) [inline docs/source/legacy-dev-docs.rst:154]: this should be opendcs not cwms. I need to sanity check but this should've been renamed as part of a branch that was recently merged it.

(also I move the dev docs out of the legacy mark as they aren't, you'll want to rebase onto the most recent master branch)

File: docs/source/legacy-dev-docs.rst [hunk 2]
```diff
@@ -213,103 +211,6 @@ To acquire the configured credentials the following can be used:
     }
     ...
 
-PropertyProvider
```
Comment (MikeNeilson) [inline docs/source/legacy-dev-docs.rst:216]: This shouldn't be removed, as above you'll just need to pull the latest changes from master in.

File: docs/source/start-decoding.rst [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+A major bulk of the examples here are likely not typically
+full raw messages from sources.  The sample messages are 
+displayed for the purposes of providing examples of how 
+the format statements function and decoding information.
+
+Further details and advanced topics on the DECODING can be 
+found in the DECODING manual ::ref DECODES Guide <./legacy-decoding-guide.rst>
+
+
+How do I launch the Screening Editor?
```
Comment (MikeNeilson) [inline docs/source/start-decoding.rst:19]: This appears to be in the wrong file.

**Review by MikeNeilson — CHANGES_REQUESTED** [review]
Looking good. Needs to keep/pickup some recent changes that are in the master branch.

File: docs/source/dev-docs.rst [hunk 4]
```diff
@@ -151,13 +151,11 @@ system you can turn pool tracing on for an application with the following java f
 
 .. code-block:: bash
 
-    DECJ_MAXHEAP="-Dopendcs.connection.pool.trace=true" routsched ...
+    DECJ_MAXHEAP="-Dcwms.connection.pool.trace=true" routsched ...
```
Comment (zamusacelre) [inline docs/source/dev-docs.rst:154]:
```suggestion
    DECJ_MAXHEAP="-Dopendcs.connection.pool.trace=true" routsched ...
```

File: docs/source/legacy-dev-docs.rst (hunk 2 above)
Comment (zamusacelre) [reply docs/source/legacy-dev-docs.rst]: This is I believe the one example of where changed/edits were made on the master. I rebased. However when I try including the content in PropertyProvider, I get an warning/error on the build.  Unsure how to proceed on this one.

**Review by MikeNeilson — APPROVED** [review]

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**